### PR TITLE
Drf & grid

### PR DESCRIPTION
**Pull request by kalbermattenm**

### CODE REVIEW

**Review by maltaesousa — APPROVED** [review]

File: parcel_historisation/serializers.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    designation = serializers.SlugRelatedField(
+        queryset=Designation.objects.all(),
+        slug_field='filename')
+
+    state = serializers.SlugRelatedField(
+        queryset=State.objects.all(),
+        slug_field='name')
+
+   # Keeping that as an example (1)
+   # roro = serializers.SerializerMethodField()
```
[the remark this answers is not in the document]
Comment (kalbermattenm) [reply parcel_historisation/serializers.py:15]: :grimacing:

File: parcel_historisation/static/parcel_historisation/parcel_historisation.js [hunk 2]
```diff
@@ -248,7 +248,125 @@ ph.resetSubmitForm = (div) => {
...
+        ]
+      ),
+      total: data => data.count,
+    },
+    pagination: {
+      limit: 20,
+      server: {
+        url: (prev, page) => {
+          page += 1;
+          if (page === 0) {
```
[the remark this answers is not in the document]
Comment (kalbermattenm) [reply parcel_historisation/static/parcel_historisation/parcel_historisation.js:317]: Nice catch!

[the remark this answers is not in the document]
Comment (kalbermattenm) [reply parcel_historisation/static/parcel_historisation/parcel_historisation.js:317]: page is never negative (I have played around with it)

[the remark this answers is not in the document]
Comment (maltaesousa) [reply parcel_historisation/static/parcel_historisation/parcel_historisation.js]: But if page is = 0, Then you add 1 to it so "page === 0" is never true, right?

[the remark this answers is not in the document]
Comment (kalbermattenm) [reply parcel_historisation/static/parcel_historisation/parcel_historisation.js]: Yes, exactly

**Pull request merged**